src/scss/components/index.scss

```diff
@@ -20,7 +20,8 @@
 // Components.
 // @import 'component-name/index';
 @import 'brand-bar/index';
-@import 'main-nav/index';
-@import 'search-bar/index';
 @import 'fat-footer/index';
 @import 'hero-banner/index';
+@import 'main-nav/index';
+@import '_secondary-nav/index';
```

Comment:
For my reference, what was the reason you added the `_` to the secondary-nav component name?

Reply:
@jdwjdwjdw That shouldn't be there. I will fix.